clients/segments/segment_test.go

```diff
@@ -269,39 +269,7 @@ func TestGetAll(t *testing.T) {
 	})
 }
 
-func TestUpsert(t *testing.T) {
-
-	t.Run("unexpected error while checking for status on server", func(t *testing.T) {
-		payload := `{
-  "uid": "qW5qn449RsG",
-  "name": "dev_environment",
-  "description": "only includes data of the dev environment",
-  "variables": {
-    "type": "query",
-    "value": "fetch logs | limit 1"
-  },
-  "isPublic": false,
-  "owner": "2f321c04-566e-4779-b576-3c033b8cd9e9",
-  "allowedOperations": [
-    "READ",
-    "WRITE",
-    "DELETE"
-  ],
-  "includes": []
-}`
-		ctx := testutils.ContextWithLogger(t)
-		mockClient := segments.NewMockclient(gomock.NewController(t))
-		mockClient.EXPECT().
-			Get(ctx, "uid", gomock.Any()).
-			Return(nil, errors.New("some unexpected error"))
-
-		fsClient := segments.NewTestClient(mockClient)
-		actual, err := fsClient.Upsert(ctx, "uid", []byte(payload))
-
-		require.Error(t, err)
-		require.Empty(t, actual)
-	})
-
+func TestCreate(t *testing.T) {
```

Comment:
Because of editing the given payload, I would like to have a test which tests (and documents) what has the precedence, given or get value for `owner` and `version`.